setup.py

```diff
@@ -63,7 +63,7 @@
     ),
     extras_require=extras,
     install_requires=[
-        "aiofiles <=0.6.0",
+        "aiofiles <=0.9.0",
```

Comment:
```suggestion
        "aiofiles <1",
```
We can pin this to <1.